src/App.test.tsx

```diff
@@ -48,11 +54,11 @@ describe("render App component", () => {
     fireEvent.change(inputText, { target: { value: "New" } });
     expect(btnAdd).not.toBeDisabled();
 
-    await act(async () => {
+    await (async () => {
       await fireEvent.click(btnAdd);
     });
```

Comment:
```suggestion
    await (async () => {
      await fireEvent.click(btnAdd);
    });
```
If we remove act we don't need any of the async awaits here.